assets/src/scripts/components/hydrograph/index.js

```diff
@@ -294,4 +363,4 @@ const attachToNode = function (node, {siteno} = {}) {
 };
 
 
-module.exports = {attachToNode, timeSeriesGraph};
+module.exports = {attachToNode, timeSeriesLegend, timeSeriesGraph};
```

Comment:
I'm not sure I like that we are exporting timeSeriesLegend and timeSeriesGraph only for testing. What do others think? I think they can be tested particularly now that we have separated the state from the rendering without exporting these functions.